app/controllers/api/v3/statements_controller.rb

```diff
@@ -1,12 +1,40 @@
 module API
   module V3
     class StatementsController < BaseController
+      include DateFilterable
+      include RegistrationPeriodFilterable
+
       def index
-        # Temp to demonstrate pagination.
-        render json: { data: paginate(Statement.all) }.to_json
+        render json: to_json(paginate(statements_query.statements))
```

Comment:
we might need to have a look at the serialiser:
```
09:50:33 web.1  |   ↳ app/serializers/statement_serializer.rb:7:in `block in <class:AttributesSerializer>'
09:50:33 web.1  |   CACHE ActiveLeadProvider Load (0.0ms)  SELECT "active_lead_providers".* FROM "active_lead_providers" WHERE "active_lead_providers"."id" = $1 LIMIT $2  [["id", 232], ["LIMIT", 1]]
```
Although its cached and could be ok not an n+1, worth revisiting incase